spec/messages.js

```diff
@@ -896,6 +896,29 @@ describe("A Chat Message", function () {
         done();
     }));
 
+    it("will remove url query parameters from hyperlinks as set",
+        mock.initConverse(
+            ['rosterGroupsFetched', 'chatBoxesFetched'], {},
+            async function (done, _converse) {
+
+        await mock.waitForRoster(_converse, 'current');
+        await mock.openControlBox(_converse);
+        const contact_jid = mock.cur_names[0].replace(/ /g,'.').toLowerCase() + '@montague.lit';
+        await mock.openChatBoxFor(_converse, contact_jid);
+        const view = _converse.api.chatviews.get(contact_jid);
+        _converse.api.settings.set('filter_url_query_params', ['utm_medium', 'utm_content', 's']);
```

Comment:
Instead of manually calling `settings.set` here, you can just pass in default configuration settings to `mock.initConverse`. The 2nd parameter takes configuration settings.